src/zjs_i2c.c

```diff
@@ -169,13 +175,13 @@ static jerry_value_t zjs_i2c_write(const jerry_value_t function_obj,
     } else {
         return zjs_error("zjs_i2c_write: missing data buffer");
     }
-    int reply;
+
     uint32_t address = (uint32_t)jerry_get_number_value(argv[0]);
 
-    uint8_t error_msg = zjs_i2c_handle_write((uint8_t)bus,
-                                             dataBuf->buffer,
-                                             dataBuf->bufsize,
-                                             (uint16_t)address);
+    int error_msg = zjs_i2c_handle_write((uint8_t)bus,
+                                         dataBuf->buffer,
+                                         dataBuf->bufsize,
+                                         (uint16_t)address);
 
     return jerry_create_number(error_msg);
```

Comment:
This should really return an error object back to JavaScript instead I think? The API documentation seems to say this function is void so it should return undefined in the success case.